src/utils/eth_transaction.cairo

```diff
@@ -110,9 +111,12 @@ namespace EthTransaction {
         let (amount) = Helpers.bytes_to_felt(
             sub_items[gas_limit_idx + 2].data_len, sub_items[gas_limit_idx + 2].data, 0
         );
+        let (nonce) = Helpers.bytes_to_felt(
```

Comment:
see comment above, can we keep the original definition ordering? update the start index definition (here `gas_limit_idx` to become `nonce_index` and use this baseline everywhere)